google-cloud-tools-plugin/src/com/google/gct/idea/debugger/ui/CloudAttachDialog.java

```diff
@@ -129,19 +129,19 @@ public void actionPerformed(ActionEvent e) {
       }
     });
 
-    myWarningLabel.setVisible(false);
-    myWarningLabel.setFont(new Font(myWarningLabel.getFont().getName(), Font.BOLD,
-                                    myWarningLabel.getFont().getSize() - 1));
-    myWarningLabel.setForeground(JBColor.RED);
-    myWarningLabel2.setVisible(false);
-    myWarningLabel2.setFont(new Font(myWarningLabel2.getFont().getName(), Font.PLAIN,
-                                     myWarningLabel.getFont().getSize() - 1));
-    myWarningLabel2.setText(GctBundle.getString("clouddebug.sourcedoesnotmatch"));
```

Comment:
actually, the warning label is overridden for several different contexts. So its not just 1 string that will always appear in it. Also, myWarningLabel was just the "title" (e.g. "Warning: ") - I changed it to warningHeader / warningMessage to make it clearer